### PR TITLE
feat: update example device files

### PR DESCRIPTION
**Pull request by ChristianDarr-personal**
**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?) N/A
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?) N/A
  <link to docs PR>

### CODE REVIEW

File: cmd/res/devices/intel.realsense.yaml.example [hunk 1]
```diff
@@ -0,0 +1,9 @@
+deviceList:
+  - name: "Intel-Realsense"
+    profileName: "USB-Camera-General"
+    description: "Example Realsense Camera"
+    labels: [ "device-usb-camera-example", ]
+    protocols:
+      USB:
+        Paths: ["/dev/video0","/dev/video2","/dev/video4",]
```
Comment (ajcasagrande) [inline cmd/res/devices/intel.realsense.yaml.example]: Would it be better to split these out using the `-` syntax?

```suggestion
        Paths:
         - "/dev/video0"
         - "/dev/video2"
         - "/dev/video4"
```

File: cmd/res/devices/general.usb.camera.yaml.example [hunk 2]
```diff
@@ -5,5 +5,5 @@ deviceList:
     labels: [ "device-usb-camera-example", ]
     protocols:
       USB:
-        Path: "/dev/video0"
+        Paths: ["/dev/video0",]
```
Comment (ajcasagrande) [inline cmd/res/devices/general.usb.camera.yaml.example]: Is the `,]` ending necessary to make Yaml turn it into an array?

**Review by ajcasagrande — APPROVED** [review]

**Review by presatish — APPROVED** [review]
LGTM

**Review by presatish — CHANGES_REQUESTED** [review]
When I tested using the document (used Paths) it was leading to some issues. So this needs to be tested. Also the doc needs change too.

**Review by presatish — APPROVED** [review]
LGTM

**Pull request merged**